src/arrayMethodReduce.js

```diff
@@ -5,7 +5,24 @@
  */
 function applyCustomReduce() {
   [].__proto__.reduce2 = function(callback, initialValue) {
-    // write code here
+    let prev = arguments < 2
```

Comment:
```suggestion
    let prev = arguments.length < 2
```

І тепер видали нижче перевірку і тести не будуть падати)
Якщо з'являться якісь питанн, то сміло звертайся в fe_chat, там зможеш швидше отримати відповідь, бо тут ми зможемо відповісти тільки коли ти попросиш повторне рев'ю